### PR TITLE
Enable performance tests on Navi3x

### PR DESCRIPTION
**Pull request by yiqian1**
See [Navi3x performance reports](http://rocmhead.amd.com:8080/job/MLIR/job/mlir/view/change-requests/job/PR-1276/21/Performance_20report_20for_20gfx1100/).

It turned out that perfRunner.py already takes care of timeout cases. There is no need to try to catch timeouts in Jenkinsfile.

### CODE REVIEW

File: mlir/utils/jenkins/Jenkinsfile [hunk 1]
```diff
@@ -112,30 +112,32 @@ void preMergeCheckPackage(String codepath) {
 
 void postProcessPerfRes(String chip) {
     publishHTML (target: [
-        allowMissing: false,
+        allowMissing: ${chip=='gfx1100' ? true, false},
```
Comment (krzysz00) [inline mlir/utils/jenkins/Jenkinsfile]: `== gfx1100` is the wrong way to check for Navi3x. You probably want `.startswith("gfx11")`

**Review by krzysz00 — COMMENTED** [review]
I'm not sure I like this approach. Wouldn't it make more sense to exclude the convolution perf testing on Navi3x cards instead of having all these catches?

Similarly, instead of `allowMissing`, modify the list of files dynamically and have it not include the convolution report on Navi3x.

File: mlir/utils/jenkins/Jenkinsfile (hunk 1 above)
Comment (krzysz00) [reply mlir/utils/jenkins/Jenkinsfile]: ... never mind, thought this was the Python script

**Comment by yiqian1** [comment]
> I'm not sure I like this approach. Wouldn't it make more sense to exclude the convolution perf testing on Navi3x cards instead of having all these catches?
> 
> Similarly, instead of `allowMissing`, modify the list of files dynamically and have it not include the convolution report on Navi3x.

Why do we want to exclude the convolution perf testing on Navi3x?

I think I can move the creation of html right after the performance test, similar to what "Test MLIR vs CK" does.

**Review by krzysz00 — APPROVED** [review]

**Pull request merged**